flexmeasures/api/v3_0/assets.py

```diff
@@ -318,3 +338,173 @@ def get_chart_data(self, id: int, asset: GenericAsset, **kwargs):
+                        "soc-unit": "kWh",
+                        "soc-targets": [
+                            {
+                                "value": 25,
+                                "datetime": "2015-06-02T16:00:00+00:00"
+                            },
+                        ],
+                        "soc-minima": {"sensor" : 300},
+                        "soc-min": 10,
+                        "soc-max": 25,
+                        "charging-efficiency": "120%",
+                        "discharging-efficiency": {"sensor": 98},
+                        "storage-efficiency": 0.9999,
+                        "power-capacity": "25kW",
+                        "consumption-capacity" : {"sensor": 42},
+                        "production-capacity" : "30 kW"
+                    },
+                ],
+                "flex-context": {
+                    "consumption-price-sensor": 9,
+                    "production-price-sensor": 10,
+                    "inflexible-device-sensors": [13, 14, 15],
+                    "site-power-capacity": "100kW",
+                    "site-production-capacity": "80kW",
+                    "site-consumption-capacity": {"sensor": 32}
+                }
+            }
+
+        **Example response**
+
+        This message indicates that the scheduling request has been processed without any error.
+        A scheduling job has been created with some Universally Unique Identifier (UUID),
+        which will be picked up by a worker.
+        The given UUID may be used to obtain the resulting schedule: see /assets/<id>/schedules/<uuid>.
+
+        .. sourcecode:: json
+
+            {
+                "status": "PROCESSED",
+                "schedule": "364bfd06-c1fa-430b-8d25-8f5a547651fb",
+                "message": "Request has been processed."
+            }
+
+        :reqheader Authorization: The authentication token
+        :reqheader Content-Type: application/json
+        :resheader Content-Type: application/json
+        :status 200: PROCESSED
+        :status 400: INVALID_DATA
+        :status 401: UNAUTHORIZED
+        :status 403: INVALID_SENDER
+        :status 405: INVALID_METHOD
+        :status 422: UNPROCESSABLE_ENTITY
+        """
+        end_of_schedule = start_of_schedule + duration
+
+        scheduler_kwargs = dict(
+            start=start_of_schedule,
+            end=end_of_schedule,
+            belief_time=belief_time,  # server time if no prior time was sent
+            flex_model=flex_model,
+            flex_context=flex_context,
+        )
+        try:
+            jobs = create_sequential_scheduling_job(
+                asset_or_sensor=asset, enqueue=True, **scheduler_kwargs
+            )
+        except ValidationError as err:
+            return invalid_flex_config(err.messages)
+        except ValueError as err:
+            return invalid_flex_config(str(err))
+
+        # todo: make a 'done job' and pass that job's ID here
```

Comment:
Technically, I guess I now implemented this todo, by putting the last job ID (that of the 'done job') in the response. However, I recall we discussed returning a full list of job IDs, so that the API user can use the existing `get_schedule` endpoint to retrieve schedules for individual sensors. In that case, the next step would be to spec such a response.

For example:

```json
{
    "schedules": [
        {
            "sensor": 1,
            "schedule": "<uuid>"
        },
        {
            "sensor": 2,
            "schedule": "<uuid>"
        }
    ]
}
```

Reply:
True, returning a list of schedules would be the way to go. That way, we can reuse the endpoint to get the schedules.